### DISCUSSION TITLE
Unread Notifications

### DISCUSSION THREAD

**Discussion by set** [community/community#13684] [Other Feature Feedback, Questions, & Ideas] [answered]
Github says "you have unread notifications" in notifications page but cannot list. I cannot see any notifications, but there is notification badge. Do you know why?

**Comment by MB221** [comment]
?

**Reply by ZnavoArdai** [reply]
its just bug or your intrnet slow

**Reply by set** [reply]
its a bug and I wanted to report

**Comment by zsmatrix62** [comment]
I got the same problem it begins about 2months ago, this is the screenshot:

![image](https://user-images.githubusercontent.com/2951879/163189922-4a4b3129-1bd7-43c5-8435-2351ce6933d3.png)

**Comment by NicestRudeGuy** [comment]
I do have this issue. I reload the page or clear cache to fix it.

**Reply by lorisleiva** [reply]
I've got the same issue and reloading / clearing the cache does not fix it for me.

**Comment by lorisleiva** [comment]
I've had the exact same issue multiple times. To **"force-clear" all notifications** I run the following command in the terminal:

```sh
gh api \
  --method PUT \
  -H "Accept: application/vnd.github+json" \
  /notifications \
  -f last_read_at='2022-06-10T00:00:00Z' \
  -F read=true 
```

Such that `last_read_at` is the current date/time in UTC.

If you're not used to running [`gh api` commands](https://cli.github.com/manual/gh_api), you'll need to provide your terminal with a Personal Access Token with the `notifications` scope.
- Go to the [Personal Access Token page](https://github.com/settings/tokens) in your settings.
- Click **Generate New Token**.
- Name it whatever you like, e.g. "Clearing Unread Notifications".
- Select the "**notifications**" scope on the table below.
- Click **Generate token** and copy the API Token displayed.
- In your terminal, make that token available to `gh api` by running `export GH_TOKEN="PASTE_YOUR_TOKEN_HERE"`.
- Now, run the command above such that `last_read_at` is the current date/time in UTC (or any date in the future).

**Reply by 18jad** [reply]
Legend, thanks <3

**Reply by Gsantomaggio** [reply]
thank you!

**Reply by ahbanavi** [reply]
Thank you, I was struggling with this bug for two months and this fixed the problem.

**Reply by qrkourier** [reply, marked as answer]
I just had to do this too! This worked for me has the GNU date string baked-in with the expected format.

```bash
gh api \
  --method PUT \
  -H "Accept: application/vnd.github+json" \
  /notifications \
  -f last_read_at="$(date -u +'%Y-%m-%dT%H:%M:%SZ')" \
  -F read=true
```

**Reply by Gravifer** [reply]
For powershell
```powershell
gh api `
   --method PUT `
   -H "Accept: application/vnd.github+json" `
   /notifications `
   -f last_read_at="$(date  -Format +'YYYY-MM-DDTHH:MM:SSZ')" `
   -F read=true
```

**Comment by jfullstackdev** [comment]
it's called ghost notifications, there are so many fix but one actually work for me,

https://github.com/community/community/discussions/6874#discussioncomment-2859125

you can look at the other answers too

**Comment by KaganCanSit** [comment]
I created a ticket on a similar situation and received the following answer. Maybe could help.

> Hi Kağan,
> The issue you are experiencing is a known bug that often occurs when you receive notifications from accounts that have been suspended or marked as spammy, or mentions that have since been deleted. Our engineering team is aware of this issue, and they are actively working to address it. While I cannot make any specific promises regarding updates or changes, please rest assured that your report is being investigated.
>  
> In the meantime, there is a workaround you can try. You can mark all notifications as using our Activity API. To do so, please run the following command, replacing with your personal access token. Note that the token needs to have scope checked.
>  
>  
> Please note that this workaround will not remove the repository from the list of repositories you have received notifications for, which appears on the left panel of the notifications page. Unfortunately, it is currently not possible to edit that list.
>  
> Give this a try and let me know how it goes.
>  
> Regards,
> Rockynoti 
> `curl -X PUT -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $TOKEN" https://api.github.com/notifications`

**Comment by KevinRJF** [comment]
have the same issue, I think that's just a bug, hopefully gets fixed soon.
